packages/nodes-base/nodes/BambooHR/v1/transport/index.ts

```diff
@@ -0,0 +1,58 @@
+import {
+	IExecuteFunctions,
+	IHookFunctions,
+	ILoadOptionsFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	IHttpRequestOptions,
+	NodeApiError,
+	NodeOperationError,
+} from 'n8n-workflow';
+
+/**
+ * Make an API request to Mattermost
+ */
+export async function apiRequest(
+	this: IHookFunctions | IExecuteFunctions | ILoadOptionsFunctions,
+	method: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'HEAD',
```

Comment:
Are they all used by your implementation?

Reply:
I removed only 'HEAD' form here since we are using GET, POST, PUT and DELETE